sycl/include/sycl/builtins.hpp

```diff
@@ -48,6 +48,8 @@ template <class T, int N> marray<T, N> to_marray(vec<T, N> x) {
 namespace __sycl_std = __host_std;
 #endif
 
+#define __SYCL_COMMA ,
```

Comment:
Please add `#undef` for it

Reply:
[f0d710f](https://github.com/intel/llvm/pull/10048/commits/f0d710f252614766a9a6f4d31a22fbd4f8c444c9)